src/v1/models/tenant.ts

```diff
@@ -0,0 +1,43 @@
+import { DataTypes } from 'sequelize';
+import { AppDataSource } from '../config';
+
+export const Tenant = AppDataSource.define(
+  'tenant',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    tenant_name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    tenant_type: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    is_active: {
+      type: DataTypes.BOOLEAN,
+      allowNull: false,
+    },
+    status: {
```

Comment:
enum ‘deleted’ ‘draft’ and ‘approved’ alone

Reply:
updated.